jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -0,0 +1,254 @@
+---
+description: Config policy management for CircleCI project configurations.
+version:
+- Cloud
+---
+= Config policy management overview
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+NOTE: Config policy management is available on the **Scale** plan and is currently in **open preview**. All aspects of the feature are subject to change.
+
+[#introduction]
+== Introduction
+
+Config policy management allows you to create policies for governing CircleCI project configurations. In its strictest case, if a project configuration does not comply with the rules set out in the associated policy, that projects pipelines cannot be triggered until it does comply.
+
+CircleCI uses `config.yml` files to define CI/CD pipelines at the project level. This is a convenient way of developing and iterating quickly, as each pipeline can be made to meet the needs of the project as it grows. However, it can be difficult to manage and enforce organization-wide conventions and security policies. Config policy management adds in this layer of control.
+
+Config policy decisions are stored, and can be audited. This provides useful data about the pipeline defintions being run within your organization.
+
+[#how-config-policy-management-works]
+== How config policy management works
```

Comment:
I'm not sure if it would be the most appropriate place, but I think I'd like to see somewhere on this page where one would go to actually set up / manage policies. I know the details are in the other pages but I would have preferred to realize earlier that the policies go in a folder in the repo, as opposed to, for example, somewhere in the CircleCI web app?

Reply:
✅

Reply:
I've added a note and a link to the page for now, but I can revisit later and add more if we think it's needed